src/main/java/io/turntabl/dataformatting/AnsiConsole.java

```diff
@@ -0,0 +1,75 @@
+package io.turntabl.dataformatting;
+
+public class AnsiConsole {
```

Comment:
Are we going to use all this colors? if not let's remove codes we don't need

Reply:
no we are not going to use all was trying to see how it will work